app/helpers/doi-form-errors.js

```diff
@@ -27,9 +27,9 @@ export function doiFormErrors([ model ]) {
         errorAttributes = errorAttributes.concat(contributor.validations.errors.mapBy('attribute'));
       });
     }
-    if (model.identifiers) {
-      model.identifiers.forEach((identifier) => {
-        errorAttributes = errorAttributes.concat(identifier.validations.errors.mapBy('attribute'));
+    if (model.alternateIdentifiers) {
+      model.alternateIdentifiers.forEach((alternateIdentifier) => {
+        errorAttributes = errorAttributes.concat(alternateIdentifier.validations.errors.mapBy('attribute'));
```

Comment:
I think it would be more useful to the user that the error displayed is the `error message` of the validation, rather than just the attribution name. That way the user can have some agency on what to do.  

example: https://github.com/datacite/bracco/blob/8f476bc5c63c329e3db2ca1a0a6d40848ee702e3/app/helpers/doi-form-errors.js#L42

Reply:
This is not how the function currently works, so it is better to discuss this in more detail at some time. The fundamental problem is of course that the current setup makes it hard to display error messages next to an input box.